src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -80,6 +80,9 @@
    :up-time-ms WholeMilliseconds
    :start-time-ms WholeMilliseconds})
 
+(def DebugLoggingConfig
+  (schema/maybe {:interval-milliseconds schema/Int}))
```

Comment:
Question - did you change this to millis mostly for the sake of the test?  I don't think a user would probably ever want to configure this for less than 1 second (it seems way more likely that it's going to be something like every 30 mins), and we should probably optimize the configuration for the user use case.

Do you know whether a float would make its way through the system?  If so, I might suggest that we change this to `interval-minutes`, and then for the test use a float < 1?